WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -139,7 +139,8 @@ def getCPUTime( CPUNormalizationFactor ):
     if not CPUNormalizationFactor:  # if CPUNormalizationFactor passed in is 0, try get it from the local cfg
       CPUNormalizationFactor = gConfig.getValue( '/LocalSite/CPUNormalizationFactor', 0.0 )
       # if CPUNormalizationFactor is not even in the local cfg, it's a problem, and yes the next line will raise an exception
-    CPUTime = CPUTime / int( CPUNormalizationFactor )
+    if int( CPUNormalizationFactor ):
```

Comment:
Yes, there is a comment about raising the exception. So, we should rely on catching it instead of checking the function return status as we normally should do. I will roll it back for the moment as it can probably change some logic elsewhere. But the exception should be rather caught inside this code and return S_ERROR